### ISSUE TITLE
Display Python results in the SASNB similar to JupyterNotebook

### ISSUE THREAD

**Issue by pestyld** [sassoftware/vscode-sas-extension#394]
When using the Python cell in the SASNB it won't display the value of a variable like Jupyter Notebook does by default. I know it might be a limitation since PROC PYTHON wraps the Python code, but I'd love the Python cells to work like they do in Jupyter so you can easily view your variable values. Not sure this is possible.

Example:
![image](https://github.com/sassoftware/vscode-sas-extension/assets/58962057/d5fa48fc-346c-4b6f-8af4-3262fdf7931d)

The simplest alternative here is to using the print function in the SASNB. But it's not as clean and takes an extra step.

**Comment by tomweber-sas** [comment]
Yes, the reason you have to use print(df.head()) in that situation is because your python code is being executed as a 'batch script' using the file parser of python. Proc Python supports both the file parser (batch execution) and the interactive parse (same as jupyter is using) or how python in a command line/shell works. So if you just typed in 
```
proc python; 
interactive; 

[ all your python code]

endinteractive;
quit;
```
instead of the tool adding `proc pyton submit;` ... `endsubmit; quit;` wrapped around your python code, then it would behave as you expect; like jupyter. I can't say I'm familiar w/ that SASNB, or if you can type SAS code in it yourself (type in the proc statement yourself), or does it only let you type in python code and it wraps everything w/ proc python? I only see one cell in the picture and I'm not clear if other cells have other SAS code in them or not. If there are different cell types, then I can see that being what's going on.
If the tool wraps the code in every cell with other 'SASisms', which is seems it would (for each cell's code), then running proc python interactively would be restricted to one cell - meaning you would have to submit the full program in one cell, and not span cells like you would in jupyter or a command prompt, because the tool would be injecting it's own 'cell wrapper' code around each cells code you supply, and that would cause issues when python tried to parse that extra SAS code. I suppose that's why they only support the 'batch' mode of execution for the proc.
